packages/hash/frontend/src/pages/[account-slug]/entities/[entity-id].page/entity-page-wrapper/entity-page-header.tsx

```diff
@@ -0,0 +1,82 @@
+import { faAsterisk } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@hashintel/hash-design-system/fontawesome-icon";
+import { Box, Stack, Typography } from "@mui/material";
+import { Container } from "@mui/system";
+import { useRouter } from "next/router";
+import slugify from "slugify";
+import { generateEntityLabel } from "../../../../../lib/entities";
+import { TopContextBar } from "../../../../shared/top-context-bar";
+import { HashOntologyIcon } from "../../../types/entity-type/hash-ontology-icon";
+import { OntologyChip } from "../../../types/entity-type/ontology-chip";
+import { useEntityEditor } from "../entity-editor-context";
+
+export const EntityPageHeader = () => {
+  const { entity } = useEntityEditor();
+  const router = useRouter();
+
+  if (!entity) {
+    return null;
+  }
+  const accountSlug = router.query["account-slug"];
```

Comment:
Here some application/route-specific data is introduced – `slug` (or `namespace`, etc) could be passed in as a prop instead, making this component usable in more contexts.